### PR TITLE
Hyundai: Car Port for Kona EV 2nd Generation (HDA2, 2023 Korean Ver.)

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Checklist**
- [x] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [x] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with stock system: `9a3aa85e86f9c3fc|2023-06-28--14-58-27`
- [x] route with openpilot: `1618132d68afc876|2023-08-27--09-32-14`
- [x] harness type: [Hyundai R](https://github.com/commaai/openpilot/wiki/Hyundai-Kia-Genesis#harness-guide)
  - https://github.com/commaai/openpilot/issues/27580
- [x] architecture: CAN-FD
- [x] message and byte/bit to suppress LFA via HDA

**Link to all physical parts during installation**
  - [comma 3X](https://comma.ai/shop/comma-3x)
  - [car harness kit](https://comma.ai/shop/car-harness)
    - [Hyundai R](https://github.com/commaai/openpilot/wiki/Hyundai-Kia-Genesis#harness-guide) (HDA2)

**Software URL**
- https://bit.ly/sp-kona-ev-2nd-gen-port

**Requires**
- https://github.com/commaai/panda/pull/1504
- https://github.com/commaai/opendbc/pull/892
- https://github.com/commaai/opendbc/pull/923

**Resolves**
- https://github.com/commaai/openpilot/issues/28724
- https://github.com/commaai/openpilot/issues/28416

Note: Hyundai Kona EV 2nd Generation is not released in the North American market until the model year 2024.

Thanks to the community Hyundai Kona EV 2nd Generation (HDA2, 2023 Korea Ver.) owner haram-KONA.

### CODE REVIEW

**Comment by sunnyhaibin** [comment]
Hyundai Kona EV 2nd Generation is not released in the North American market until the model year 2024.

**Comment by sunnyhaibin** [comment]
- New steering message: https://github.com/commaai/openpilot/issues/28831

**Comment by sunnyhaibin** [comment]
Something went wrong with my local env, squashing and force-pushing the port.

**Comment by haram-KONA** [comment]
" Hyundai: Car Port for Kona EV 2nd Generation (2023 Korean Ver.) #28725 "
No progress on this for a long time..... 
Please save "2023 KONA EV", do I have to keep disabling HDA2?
I am very sorry to have to say this to those who worked so hard to prepare and host the "Comma_Con 2023".

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -81,6 +82,7 @@ class CAR:
   KONA = "HYUNDAI KONA 2020"
   KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
   KONA_EV_2022 = "HYUNDAI KONA ELECTRIC 2022"
+  KONA_EV_2ND_GEN = "HYUNDAI KONA 2ND GEN"
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:85]:
```suggestion
  KONA_EV_2ND_GEN = "HYUNDAI KONA ELECTRIC 2ND GEN"
```

File: selfdrive/car/torque_data/override.yaml [hunk 2]
```diff
@@ -57,6 +57,7 @@ KIA CARNIVAL 4TH GEN: [1.75, 1.75, 0.15]
 GMC ACADIA DENALI 2018: [1.6, 1.6, 0.2]
 LEXUS IS 2023: [2.0, 2.0, 0.1]
 KIA SORENTO HYBRID 4TH GEN: [2.5, 2.5, 0.1]
+HYUNDAI KONA 2ND GEN: [2.86, 2.5, 0.0969]
```
Comment (sshane) [inline selfdrive/car/torque_data/override.yaml:60]:
```suggestion
HYUNDAI KONA 2ND GEN: [2.5, 2.5, 0.1]
```

from `1618132d68afc876|2023-08-30--10-18-13`
`latAccelFactorRaw = 2.4725506, frictionCoefficientRaw = 0.101607`

**Review by sshane — APPROVED** [review]

**Comment by sshane** [comment]
Nice job on the port and finding the new message to suppress LFA!

**Pull request merged**